staff/RominaFior/css-layout-simple-adaptive/styles/main.css

```diff
@@ -0,0 +1,2 @@
+@import "mobilemain.css" screen and (max-width: 768px);
+@import "desktopmain.css" screen and (min-width:769px)
```

Comment:
There are common CSS properties in both versions (desktop and mobile), these should go here to be applied always.

Reply:
Thanks for the tip 👍